modules/dnn/src/darknet/darknet_io.cpp

```diff
@@ -425,6 +425,7 @@ namespace cv {
                     }
 
                     shortcut_param.set<std::string>("op", "sum");
+                    shortcut_param.set<std::string>("output_channels_mode", "input_0_truncate");
```

Comment:
According to https://github.com/opencv/opencv/pull/15739#issuecomment-544036806, Darknet supports two modes of summation, `ELTWISE_CHANNNELS_INPUT_0` and `ELTWISE_CHANNNELS_INPUT_0_TRUNCATE` in our terms. Can we just have a single `ELTWISE_CHANNNELS_INPUT_0` that means output number of channels is the same as first input's number of channels.

There is a code which can swap inputs to have unified order of inputs: https://github.com/opencv/opencv/pull/15739/files#diff-7ac73ff12c29882cb913b6f09da2f82cR179-R185